tools/docker-compose/supervisor.conf

```diff
@@ -8,31 +8,25 @@ command = make dispatcher
 autorestart = true
 stopasgroup=true
 killasgroup=true
-stdout_logfile=/dev/stdout
-stdout_logfile_maxbytes=0
-stderr_logfile=/dev/stderr
-stderr_logfile_maxbytes=0
+stdout_events_enabled = true
+stderr_events_enabled = true
```

Comment:
Testing this change locally, this seems to keep supervisord logs like

```
2022-11-09 09:34:47,046 INFO spawned: 'fail-slowly' with pid 1258318
```

but stops sending the subprocess standard out to the supervisord standard out. For docker-compose I don't think that's what we want? Anyway, this is fairly unrelated to the feature intent here.

Reply:
Hmm I am not sure I'm following. The intent here is to send all logs to stdout in our dev env rather than /var/log/supervisor/.

Reply:
There are logs from supervisor (like what I pasted) and standard out from the processes that supervisord is managing. Yes, we're on the same page that both types of logs should go to stdout in the dev environment. My comment was saying that standard out from managed processes (like dispatcher/uwsgi/etc.) do not with those settings. When it says "stdout_events_enabled", I think it means supervisor events. The docs mention something about capture mode playing a role, but I don't understand it.